deploy/ncsa/docker-compose.yml

```diff
@@ -12,6 +12,33 @@ x-service: &service
       max-file: "5"
       max-size: "10m"
 
+x-base-producer-env: &base-producer-environment
+  LSST_DDS_PARTITION_PREFIX: ${LSST_DDS_PARTITION_PREFIX}
+  WEBSOCKET_HOST: ${LOVE_PRODUCER_WEBSOCKET_HOST}
+  PROCESS_CONNECTION_PASS: ${PROCESS_CONNECTION_PASS}
+  LOVE_CSC_PRODUCER: Test.1
+
+x-base-producer: &base-producer
+  image: ts-dockerhub.lsst.org/love-producer:${cycle}
+  environment:
+    <<: *base-producer-environment
+  networks:
+    love:
+  ipc: host
+  pid: host
+  volumes:
+    - /tmp/docker_tmp/:/tmp/
+    - ./config:/usr/src/love/producer/config/
+    - ../SALSubsystems.xml:/home/saluser/repos/ts_xml/sal_interfaces/SALSubsystems.xml
```

Comment:
I really don't like this. We mirroring another file that is being tracked elsewhere. This is a problem waiting to bite us.

Reply:
good catch... this should have been removed...

Reply:
This also means removing the file from the repo since it is not necessary.